Expecto.Differ/Library.fs

```diff
@@ -0,0 +1,45 @@
+module Expecto.Differ
+open System
+open DiffPlex.DiffBuilder
+open DiffPlex.DiffBuilder.Model
+
+let colorizedDiff (colorizeText: ConsoleColor -> string -> string) first second =
+  let first, second =
+    match box first, box second with
+    | (:? string as f), (:? string as s) ->
+      string f, string s
+    | f, s ->
+      sprintf "%A" f, sprintf "%A" s
+
+  let differ = SideBySideDiffBuilder(DiffPlex.Differ())
+  let diff = differ.BuildDiffModel(first, second)
+
+  let coloredText typ text =
+    match typ with
+    | ChangeType.Inserted -> colorizeText ConsoleColor.Green text
+    | ChangeType.Deleted -> colorizeText ConsoleColor.Red text
+    | ChangeType.Modified -> colorizeText ConsoleColor.Blue text
+    | ChangeType.Imaginary -> colorizeText ConsoleColor.Yellow text
+    | ChangeType.Unchanged | ChangeType.Imaginary | _ -> text
+
+  let colorizedDiff (lines: DiffPiece seq) =
+    lines
+    |> Seq.map (fun line ->
+      let styledLineNumber =
+        match line.Position |> Option.ofNullable with
```

Comment:
Single arguments are applied without `|>`.

Reply:
I can change it, but personally I prefer piping in this situation because then the most important thing, `line.Position`, comes first, and then the conversion from Nullable<->Option is more of a detail.